src/components/ProjectSettingsTabs/VirtualWorkstationTab/CommandRow.tsx

```diff
@@ -0,0 +1,29 @@
+import styled from "@emotion/styled";
+import { SpruceFormProps } from "components/SpruceForm";
```

Comment:
Was there a ticket to move these files to a more page specific directory? I remember us discussing it a while ago.

Reply:
Yes! EVG-16542 I'll pull it into next sprint 👍